### ISSUE TITLE
Attach a non file buffer to a project

### ISSUE THREAD

**Issue by pierrelegall** [bbatsov/projectile#1834]
This is not a bug report, but maybe a feature proposal.

I'm looking for a way to attach a (non file) buffer to a project. In my case, a vterm buffer, in the same way magit buffers are linked to their project.

I go through the source but without success. Do you have some thought about how I could achieve this?

**Comment by bbatsov** [comment]
Non-file buffers are already treated as project buffers when their default-directory is inside the project (see projectile-project-buffer-p), so a shell or term buffer started in the project gets picked up. Closing as answered.
